api/v1/__init__.py

```diff
@@ -0,0 +1,180 @@
+from pathlib import Path
+from typing import Any, Dict, Tuple
+
+import requests
+import structlog
+import yaml
+from fastapi import APIRouter, BackgroundTasks, HTTPException
+from git.repo import Repo
+from jsonschema import validate
+from jsonschema.exceptions import ValidationError
+from sqlmodel import Session
+
+from apps.backport.datasync.datasync import upload_gzip_dict
+from etl import config, paths
+from etl import grapher_model as gm
+from etl.command import main as etl_main
+from etl.db import get_engine
+from etl.files import yaml_dump
+from etl.helpers import read_json_schema
+from etl.metadata_export import merge_or_create_yaml, reorder_fields
+from etl.paths import SCHEMAS_DIR
+
+from .. import utils
+from .schemas import Indicator, UpdateIndicatorRequest
+
+log = structlog.get_logger()
+
+
+engine = get_engine()
+
+DATASET_SCHEMA = read_json_schema(path=SCHEMAS_DIR / "dataset-schema.json")
+
+
+v1 = APIRouter()
+
+
+@v1.get("/v1/health")
+def health() -> dict:
+    return {"status": "ok"}
+
+
+@v1.put("/v1/indicators/{indicator_id}")
```

Comment:
The ETL should probably be working on ETL paths by now, which is something we already know on the `owid-grapher` side. Let's do an update pattern based on those paths rather than based on MySQL ids.

Either the path can go into the URL or it can go into the PUT body, I don't have a strong preference.

I understand the downside being that you want to do pushes to R2, which are still id-based. Let's chat about it.

Reply:
I had to add it to the body because catalog path contains `/` and `#` (e.g. `grapher/agriculture/2023-04-21/uk_long_term_yields/uk_long_term_yields#oats_yield`). It's not a big deal now.